src/Processors/Transforms/buildPushingToViewsChain.cpp

```diff
@@ -286,12 +286,7 @@ Chain buildPushingToViewsChain(
         std::unique_ptr<ThreadStatus> view_thread_status_ptr = std::make_unique<ThreadStatus>();
         /// Copy of a ThreadStatus should be internal.
         view_thread_status_ptr->setInternalThread();
-        /// view_thread_status_ptr will be moved later (on and on), so need to capture raw pointer.
-        view_thread_status_ptr->deleter = [thread_status = view_thread_status_ptr.get(), running_group]
```

Comment:
Why is this line removed?

Reply:
There is only one deleter function for all cases.
```
thread_status->detachQuery();
```
Was no point to set it explicitly. I removed all those assignments.